webui/react/src/pages/ExperimentList.tsx

```diff
@@ -94,20 +210,46 @@ const ExperimentList: React.FC = () => {
             prefix={<Icon name="search" size="small" />}
             onChange={handleSearchChange} />
           <div className={css.filters}>
-            <Toggle prefixLabel="Show Archived" onChange={handleArchiveChange} />
+            <Toggle
+              checked={filters.showArchived}
+              prefixLabel="Show Archived"
+              onChange={handleArchiveChange} />
             <StateSelectFilter
               showCommandStates={false}
               value={filters.states}
               onChange={handleStateChange} />
             <UserSelectFilter value={filters.username} onChange={handleUserChange} />
           </div>
         </div>
+        <TableBatch message="Apply batch operations to multiple experiments." show={showBatch}>
+          <Button onClick={(): Promise<void> => handleBatchAction(Action.OpenTensorBoard)}>
+              Open TensorBoard
+          </Button>
+          <Button
+            disabled={!hasActivatable && false}
+            onClick={(): void => handleConfirmation(Action.Activate)}>Activate</Button>
+          <Button
+            disabled={!hasPausible}
```

Comment:
nit: pausable ?  https://www.yourdictionary.com/pausable not sure if it's a word but this seems closer

Reply:
good catch on the spelling!